src/ConnectedMode/Persistence/SolutionBindingRepository.cs

```diff
@@ -18,97 +18,94 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using System.Diagnostics;
 using SonarLint.VisualStudio.ConnectedMode.Binding;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.Binding;
 
-namespace SonarLint.VisualStudio.ConnectedMode.Persistence
+namespace SonarLint.VisualStudio.ConnectedMode.Persistence;
+
+[Export(typeof(ISolutionBindingRepository))]
+[Export(typeof(ILegacySolutionBindingRepository))]
+[PartCreationPolicy(CreationPolicy.Shared)]
+internal class SolutionBindingRepository : ISolutionBindingRepository, ILegacySolutionBindingRepository
```

Comment:
The interface here is the only actual change. Other changes in the diff are caused by the cayc cleanup